### ISSUE TITLE
Getting URL of iframe page

### ISSUE THREAD

**Issue by dualaudi** [cedaro/gravity-forms-iframe#57]
Is there any way to pass the url of the iframe page in the body of gravity form? We usually do this with {embed_url} along with {ip} - ip works obviously, but not the url embed - It shows the url of the site hosting the gravity form - not where the iframe lives.

**Comment by chillpilllike** [comment]
Hi, i got a solution for this issue,
https://autdot.com/getting-the-url-correctly/

**Comment by dualaudi** [comment]
YES!!!!! that works.. It seems to work without the function code as well. This is how I set it up [https://imgur.com/a/gWO5SPY](url)

Thank you
